libfwupdplugin/fu-bios-attrs.c

```diff
@@ -493,9 +493,22 @@ fu_bios_attrs_get_pending_reboot(FuBiosAttrs *self, gboolean *result, GError **e
 	return TRUE;
 }
 
+static gboolean
+fu_bios_attr_trusted(FwupdBiosAttr *self, gboolean trusted)
```

Comment:
how come this is here and not in `libfwupd/fwupd-bios-attr.c`? I think it's a bit odd to have `fu_bios_attrs_to_variant` and `fwupd_bios_attr_to_variant` returning different results when !trusted.

Reply:
moved it